safe_autonomy_sims/gym/inspection/sixdof_inspection_v0.py

```diff
@@ -422,8 +465,8 @@ def _init_sim(self):
                 phi=self.np_random.uniform(-np.pi / 2, np.pi / 2),
                 theta=self.np_random.uniform(0, 2 * np.pi),
             ),
-            fov=np.pi,
-            focal_length=1,
+            fov=1.0471975511965976,  # 60 degrees
```

Comment:
Can we express this in terms of pi for precision and readability? I believe it's pi/3

Reply:
You got it. changed it in both 6dof and multiagent 6dof envs.